_boards_cm/retro-dreamer-g4a-cm4.md

```diff
@@ -0,0 +1,12 @@
+---
+layout: board
+title: "Retro Dreamer G4A CM4"
+picture: "/images/board-retro-dreamer-g4a.jpg"
+functionality: "Currently Testing"
```

Comment:
This layout is for `card`—can you drop the functionality/driver_required, and add a short_description?

Reply:
@geerlingguy Absolutely! Which card type does this one fall under?

Reply:
I think I meant the template at the time was for a card, but it should be the board template, heh.